### PR TITLE
Install libtim-vx.so to lib64 if build for aarch64

### PR DESCRIPTION
**Pull request by sunshinemyson**
Signed-off-by: xiang.zhang <xiang.zhang@verisilicon.com>

### CODE REVIEW

File: src/tim/vx/tensor.cc [hunk 1]
```diff
@@ -200,7 +200,7 @@ bool TensorImpl::Init() {
         &attr,
         fd_ != -1 ? (uint8_t*)fd_ : nullptr);  // and cannot be set to const
 #else
-    if (-1 == fd) {
+    if (-1 == fd_) {
```
Comment (robert-kalmar) [inline src/tim/vx/tensor.cc:203]: Please do this as separate commit. It does not relate to libtim-vx.so installation.

**Review by thezha — APPROVED** [review]
